src/commands/functions/invoke.js

```diff
@@ -41,7 +41,8 @@ class FunctionsInvokeCommand extends Command {
       process.exit(1)
     }
 
-    let settings = await serverSettings(Object.assign({}, config.dev, flags))
+    // Pass in console.log as the logger for now.
+    let settings = await serverSettings(config.dev, flags, console.log)
```

Comment:
I could be wrong on this; but it didn't appear that `config.dev` and `flags` should be getting merged here and passed as the `devConfig` argument to `serverSettings` when it takes in `flags` as the second argument. Also, we'll get an NPE with nothing passed for the third argument (`log`).

Also, it appears there will be some conflicts with https://github.com/netlify/cli/pull/843.